test/cpp/asyncprogressworkerstream.cpp

```diff
@@ -0,0 +1,83 @@
+/*********************************************************************
+ * NAN - Native Abstractions for Node.js
+ *
+ * Copyright (c) 2016 NAN contributors
+ *
+ * MIT License <https://github.com/nodejs/nan/blob/master/LICENSE.md>
+ ********************************************************************/
+
+#ifndef _WIN32
+#include <unistd.h>
+#define Sleep(x) usleep((x)*1000)
+#endif
+#include <nan.h>
+
+using namespace Nan;  // NOLINT(build/namespaces)
+
+// custom data type. This servers as an example of how external
+// libraries could be hooked in, populate their objects and send them to JS
+struct data_t {
+  int index;
+  int data;
+};
+
+// unlike test ayyncprogressworker.cpp this test is explictely templated
```

Comment:
s/explictely/explicitly/.  I'll stop pointing it out from now on but can you capitalize and punctuate comments?